### ISSUE TITLE
Module doesn't auto-install from module.json

### ISSUE THREAD

**Issue by Azzurite** [Sky-Captain-13/foundry#1]
Download URL has to be `https://github.com/Sky-Captain-13/foundry/blob/master/deselection/deselection.zip?raw=true`

**Comment by Sky-Captain-13** [comment]
Yeah, I know. This was created way before auto updating and downloading was a thing.
